src/background/services/web3/models.ts

```diff
@@ -4,4 +4,7 @@ export enum WalletExtensionType {
   UNKNOWN = 'UNKNOWN',
   RABBY = 'RABBY',
   COINBASE = 'COINBASE',
+  PHANTOM = 'PHANTOM',
```

Comment:
I can't really make Phantom appear, I guess they don't announce themselves? 🤔

Reply:
I'm checking that when I get home later today.

Reply:
![image](https://github.com/user-attachments/assets/4d70d5c1-b3de-4a09-b1bf-52103453c871)
It seems it can work :D